### PR TITLE
db_mysql example and doco updates

### PR DESCRIPTION
**Pull request by jlp765**
First attempt at adding an example.

Should probably be same or similar added to the other db_xxxxxx modules,
hopefully as similar as possible.  I haven't attempted this.

Also, an update to fastRows() iterator to describe what happens if you break 
the iterator - just trying to be clearer on what you had about it being dangerous.

### CODE REVIEW

**Comment by Araq** [comment]
Please rebase. Sorry.

**Comment by dom96** [comment]
Could we also get examples for the other db_* modules?

**Comment by dom96** [comment]
Thank you btw :)

File: lib/impure/db_mysql.nim [hunk 1]
```diff
@@ -7,8 +7,39 @@
...
+##  import db_mysql, math
+##
+##  let theDb = open("localhost", "nim", "nim", "test")
+##
+##  theDb.exec(sql"Drop table if exists myTestTbl")
+##  theDb.exec(sql("create table myTestTbl (" &
+##      " Id    INT(11)     NOT NULL AUTO_INCREMENT PRIMARY KEY, " &
+##      " Name  VARCHAR(50) NOT NULL, " &
+##      " i     INT(11), " &
+##      " f     DECIMAL(18,10))"))
```
Comment (Xe) [inline lib/impure/db_mysql.nim]: Multi-line strings are a thing fwiw:

```nimrod
theDb.exec("""create table myTestTbl (
 Id    INT(11)     NOT NULL AUTO_INCREMENT PRIMARY KEY,
 Name  VARCHAR(50) NOT NULL,
 i     INT(11),
 f     DECIMAL(18,10));""".sql)
```

**Pull request merged**